torch/_dynamo/variables/builtin.py

```diff
@@ -1338,6 +1339,29 @@ def _call_int_float(self, tx: "InstructionTranslator", arg):
     call_int = _call_int_float
     call_float = _call_int_float
 
+    def call___build_class__(self, tx, *args, **kwargs):
+        def fail(args, kwargs):
+            unimplemented_v2(
+                gb_type="invalid call to __build_class__",
+                context=f"invalid args to __build_class__: {args} {kwargs}",
+                explanation="Encountered TypeError when trying to handle op __build_class__",
+                hints=[*graph_break_hints.DYNAMO_BUG],
```

Comment:
Could this be a USER_ERROR or SUPPORTABLE? (because we could graph break here from `check_constant_args(...) == False`.

You can do `from_exc=exc` to forward the exception object too.